src/jgikbase/idmapping/core/user.py

```diff
@@ -1,18 +1,42 @@
 """
+ID Mapping system user classes.
 
-@author: gaprice@lbl.gov
 """
+from jgikbase.idmapping.util.util import not_none
 
-# TODO NOW
-# TODO Users need to be scoped to an auth source
+
+class Authsource:
+    """
+    An authorization source for a user.
```

Comment:
What does authorization source mean? Is it the URL of a server? Is it a token or a type of token? Is it authorization or authentication?

Reply:
Fixed